BUMBAIYA/amitchauhan-v2
# Added the dynamic rendering of duotoned image feature

Added a duotoned image effect feature for the about section, where images are processed and displayed with custom light and dark colors for dark and light themes.

### Changes:
- Added DuotoneImage component.
- Integrated it into the homepage to process portfolio images dynamically.

### How to Test:
- Visit the homepage and verify that the images are rendered with the duotone effect.
- Test the site on both light and dark themes.
